Gemfile

```diff
@@ -6,3 +6,4 @@ gem 'slather',
   :branch   => "feature-profdata"
 gem 'xcpretty'
 gem 'xcpretty-travis-formatter'
+gem 'carthage'
```

Comment:
Don’t think there’s a carthage gem? You install it via Homebrew. Also Travis-CI has it installed by default.

We might want to make sure it’s the latest though, so in our `ci/before_install.sh` script add:
```shell
brew update
brew install carthage
```